lib/csv_export.rb

```diff
@@ -70,4 +70,20 @@ def download_equipment_data
     download_zip([categories, models, items],
                  "EquipmentData_#{Time.zone.now.to_s(:number)}")
   end
+
+  # NOTE: this method depends on ActionController
+  def download_category_data(cat)
+    categories = [[cat], %w(name max_per_user max_checkout_length
+                            max_renewal_times max_renewal_length
+                            renewal_days_before_due sort_order)]
+    models = [cat.equipment_models.all, %w(category name description late_fee
+                                           replacement_fee max_per_user
+                                           max_renewal_length)]
+
+    eq_models = cat.equipment_models.all.map(&:id)
```

Comment:
Does this call a second query? We're already calling `cat.equipment_models.all` on line 79 so we should probably make sure it's only happening once.